.github/workflows/validate.yml

```diff
@@ -39,6 +39,16 @@ jobs:
         with:
           version: v1.54
 
+  codespell:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
+    - name: install deps
+      # Version of codespell bundled with Ubuntu is way old, so use pip.
+      run: pip install codespell
+    - name: run codespell
+      run: codespell
+
```

Comment:
Oh! I wanted to check if there was an action for this; and  then I stumbled on a PR that was adding it; so: yes, there's an action for this; https://github.com/codespell-project/actions-codespell so we may as well use that?